code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -528,24 +528,24 @@
 	if(istype(passed_mob, /mob/living/carbon/human))
 		var/mob/living/carbon/human/passed_human = passed_mob
 		if (issynth(passed_human))
-			return 1
+			return TRUE
 		switch(hudtype)
 			if("security")
 				if(skillcheck(passed_human, SKILL_POLICE, SKILL_POLICE_SKILLED))
 					var/datum/mob_hud/sec_hud = GLOB.huds[MOB_HUD_SECURITY_ADVANCED]
-					if(locate(passed_mob) in sec_hud.hudusers)
+					if(sec_hud.hudusers[passed_human])
 						return TRUE
 			if("medical")
 				if(skillcheck(passed_human, SKILL_MEDICAL, SKILL_MEDICAL_MEDIC))
 					var/datum/mob_hud/med_hud = GLOB.huds[MOB_HUD_MEDICAL_ADVANCED]
-					if(locate(passed_mob) in med_hud.hudusers)
+					if(med_hud.hudusers[passed_human])
 						return TRUE
 			if("squadleader")
 				var/datum/mob_hud/faction_hud = GLOB.huds[MOB_HUD_FACTION_USCM]
 				if(passed_human.mind && passed_human.assigned_squad && passed_human.assigned_squad.squad_leader == passed_human && locate(passed_mob) in faction_hud.hudusers)
```

Comment:
We should fix this one too then.
```suggestion
				if(passed_human.mind && passed_human.assigned_squad && passed_human.assigned_squad.squad_leader == passed_human && faction_hud.hudusers[passed_mob])
```

Reply:
yeah my bad, thought I already added them.